### DISCUSSION TITLE
Depreciation and depreciation schedule

### DISCUSSION THREAD

**Discussion by ChrisQGeorge** [sysadminsmedia/homebox#744] [Ideas]
### What is the problem you are trying to solve with this feature?

The value of fixed assets in the system remains at the purchased price indefinitely without manually editing the prices. This leads to a situation where you have to choose between having an incorrect item value and the correct purchase price, or the correct value, but lose the purchase price information. This leads to a situation where you have to choose between having an incorrect item value and the correct purchase price or the correct value but losing the purchase price information.

### What is the solution you are proposing?

The ability to depreciate the value of an asset with some fixed ruleset or depreciation schedule. E.g., if an asset's useful lifespan is five years and valued at $5000, its value would decrease by $83.34 every month if using the straight-line method.

### What alternatives have you considered?

An additional attribute could be created to hold the original price, regular exports could be run on the whole inventory, and a Python script could be created to edit the exported CSV manually, but all the same logic that would go into making this script could also be used to implement this feature.

### Contributions

- [X] I have searched through existing issues and feature requests to see if my idea has already been proposed.
- [X] If this feature is accepted, I would be willing to help implement and maintain this feature.
- [X] If this feature is accepted, I'm willing to sponsor the development of this feature.

**Comment by a2brew** [comment]
Love this idea!

**Comment by Ryan-597** [comment]
For my own purposes however I don't need the monetary value, I'm only looking for the depreciation schedule with a notification if possible. For example item was bought on _date_ will depreciate in _5 years_. So I'll be notified in time when the item needs to be replaced. Great comment ChrisQGeorge

**Comment by tankerkiller125** [comment]
Please note we're converting this issue into a discussion as per our new procedures/policies. You can read more about that in our announcement #712

**Comment by mmPaumard** [comment]
I upvote this :)
A good first step would be to **add a field for the annual depreciation rate**, then the user could easily export and run the calculation. That sounds like a very quick update to me, so I hope it can be done quickly :)
Subsequently, the day-to-day valuation can be internalized, e.g., display the current value of the object

**Reply by stmille** [reply]
Likewise, I'd go so far as to say that without accounting for depreciation tracking the value of assets is not actually possible currently. This is a very important feature.
